test/jdk/javax/swing/border/Test4129681.java

```diff
@@ -21,52 +21,63 @@
  * questions.
  */
 
-import java.awt.event.ItemEvent;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Point;
 import javax.swing.BorderFactory;
-import javax.swing.Box;
-import javax.swing.JCheckBox;
-import javax.swing.JComponent;
 import javax.swing.JLabel;
+import javax.swing.UIManager;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
 
 /*
  * @test
  * @bug 4129681
  * @summary Tests disabling of titled border's caption
- * @library /java/awt/regtesthelpers
- * @build PassFailJFrame
- * @run main/manual Test4129681
+ * @run main Test4129681
  */
 
 public class Test4129681 {
     public static void main(String[] args) throws Exception {
-        String testInstructions = """
-                Click the checkbox to disable the label.
-                The test passes if the title of the border
-                is disabled as well as the label.
-                """;
+        int correctColoredPixels = 0;
+        int totalPixels = 0;
+        int tolerance = 20;
+        JLabel label;
+        Color labelDisableColor = Color.RED;
+        Dimension SIZE = new Dimension(100, 40);
+        Point startPoint = new Point(8, 4);
+        Point endPoint = new Point(18, 14);
 
-        PassFailJFrame.builder()
-                .title("Test Instructions")
-                .instructions(testInstructions)
-                .rows(4)
-                .columns(25)
-                .splitUI(Test4129681::init)
-                .build()
-                .awaitAndCheck();
-    }
+        UIManager.setLookAndFeel("javax.swing.plaf.metal.MetalLookAndFeel");
+        System.setProperty("sun.java2d.uiScale", "1.0");
```

Comment:
If you are going to do this in code it needs to be line 1 of main before we call any java.desktop code.

You can also pass it on the command line of the test
@run main -Dsun.java2d.uiScale=1 Test4129681

Reply:
Updated.